test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGet.CommandLine.Test.csproj

```diff
@@ -35,7 +35,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build.Framework" ExcludeAssets="runtime" />
```

Comment:
Upgrading the VSSDK version introduced a bunch of package downgrade warnings, but because of "treat warnings as errors", they were breaking the build. Since they're all test projects, I hope the removal of `ExcludeAssets="runtime"` doesn't cause any problems, but tests have passed (except VS tests failing since we're running on 17.4, but need a 17.5 API, and what appears to be just a "normal" flaky test).

Reply:
Ah that makes sense.  It should be safe since this package is pulled in transitively by `Microsoft.Build` and it has `ExcludeAssets="runtime"`